### PR TITLE
WIP CI for networking components

### PR DESCRIPTION
**Pull request by cybertron**
This isn't working yet, but it provides a basic outline of what I think we need to be able to do in our CI runs. I'm pushing it as a WIP so I have something to discuss with the team and figure out how to move forward.

### CODE REVIEW

File: 06_create_cluster.sh [hunk 1]
```diff
@@ -57,11 +60,16 @@ fi
...
 if [ -n "$IMAGE_OVERRIDES" ]; then
-    # This is consistently failing for me for some reason. It seems to be
-    # completing anyway though...
+    # This is what I would like to do, but it doesn't work
+#     oc adm release new -n ocp \
+#         --server https://api.ci.openshift.org \
+#         --from-release "$OPENSHIFT_RELEASE_IMAGE" \
+#         --to-image quay.io/cybertron/origin-release:v4.2 \
+#         $IMAGE_OVERRIDES || :
+    # This works in isolation but not when run as part of this script. :-/
```
Comment (hardys) [inline 06_create_cluster.sh]: cc @derekhiggins @stbenjam - can we use this as an alternative to https://github.com/openshift-metal3/dev-scripts/pull/759 and https://github.com/openshift-metal3/dev-scripts/pull/719 ?

It seems like a common approach here for all potentially overridden container images would be good, and using a custom release probably aligns better with integration into openshift CI?

Comment (imain) [reply 06_create_cluster.sh]: Yeah I think this would be super helpful for testing a lot of new stuff.

Comment (derekhiggins) [reply 06_create_cluster.sh]: I've closed 719, not sure how I ended up with two patch doing similar stuff...

I wonder if we should include some way of using a local registry (similar to https://github.com/openshift-metal3/dev-scripts/pull/759) to save bandwidth? I'm currently looking into integrating into the openshift build system which may eventually provide an alternative.

Comment (cybertron) [reply 06_create_cluster.sh]: Last time I checked, there wasn't really a way to use a local registry with the installer. There's no way to tell it to use --tls-verify=false when retrieving images, so it barfs on the "insecure" registry.

I would dearly love it if we could start using a local registry though. The amount of bandwidth and time that I burn from repeatedly pulling the same images is absurd. :-/

Comment (derekhiggins) [reply 06_create_cluster.sh]: > Last time I checked, there wasn't really a way to use a local registry with the installer. There's no way to tell it to use --tls-verify=false when retrieving images, so it barfs on the "insecure" registry.
> 
> I would dearly love it if we could start using a local registry though. The amount of bandwidth and time that I burn from repeatedly pulling the same images is absurd. :-/

I couldn't find a way to do it either without hacking the code, see the PR I linked to above
https://github.com/openshift-metal3/dev-scripts/pull/759/files#diff-13094614d976ac6f9d0cc2f87e41f332R36

**Comment by cybertron** [comment]
This has been addressed elsewhere. No need to keep the PR open.

**Pull request closed without merging**